share/management/commands/forceingest.py

```diff
@@ -12,7 +12,7 @@
 from share.util.osf import osf_sources
 
 
-MAX_RETRIES = 10
+MAX_RETRIES = 128
```

Comment:
🤔

Reply:
More accurate would be `MAX_MERGES` -- it'll merge one thing, then retry, then merge one more thing, then retry... if there are 30 contributors that need merging, want to handle it.

Also this is now an interactive script that I babysit, so easy to skip if it looks wrong.

Reply:
That makes more sense 👍